Further clarify reporting of security issues (#83)
apache/solr-site: content/pages/security.md

## How to report a security issue

### Published CVEs Detected by Scanners
Every CVE that is detected by a software scanner is by definition already public knowledge. That means the Solr PMC and the rest of the world probably already know about it.

To find a path forward in addressing a detected CVE we suggest the following process for fastest results:

1. Check further down this page to see if the CVE is listed as exploitable in Solr.
2. Check the [officially published non-exploitable vulnerabilities](https://cwiki.apache.org/confluence/display/SOLR/SolrSecurity#SolrSecurity-SolrandVulnerabilityScanningTools) list to see if the CVE is listed as not exploitable in Solr
3. Search through the [Solr users mailing list archive](https://lists.apache.org/list.html?users@solr.apache.org)  to see if anyone else has brought up this dependency CVE.
4. If no one has, then please do [subscribe to the users mailing list](https://solr.apache.org/community.html#mailing-lists-chat) and then send an email asking about the CVE.

> ### Dos and Don'ts

Reviewer: ```suggestion
#### Dos and Don'ts
```

Sub section maybe?

* Please DO discuss the possible need for library upgrades on the user list. 
* Please DO search Jira for the CVE number to see if we are addressing it already.
* Please DO create Jira issues and associated pull requests to propose and discuss upgrades of *a single specific* dependency.
* Please DO NOT attach a scan report, or paste output of a scan into Jira (just link the CVE instead)
* Please DO NOT email the security email below with a scan report it will be ignored.

> ### Use of Jira

Reviewer: ```suggestion
#### Use of Jira
```

Again, maybe a subsection of the dependency CVEs?

Jira is for discussing specific development modifications. Any Jira that contains only scan report output, or references multiple dependencies at the same time is likely to be ignored/closed. The large number of folks sending us reports of things that are already known is a serious drag on our (volunteer) time so **please search Jira** before opening a new issue. 

### New Exploits <span style="color:blue">You</span> Discover in Solr

> The Solr PMC greatly appreciates the reporting of new security vulnerabilities found in Solr itself or demonstrations of exploit vulnerability via dependencies. **It is important not to publish a previously unknown exploit**, or exploit demonstration code on public mailing lists. Please disclose new exploits responsibly by following these [ASF guidelines](https://www.apache.org/security/) for reporting. The contact email for reporting newly discovered exploits in Solr is <mailto:security@solr.apache.org>.

Reviewer: ```suggestion
The Solr PMC greatly appreciates reports of new security vulnerabilities found in Solr itself or demonstrations of exploiting vulnerabilities via dependencies.
**It is important not to publish a previously unknown exploit**, or exploit demonstration code on public mailing lists.
Please disclose new exploits responsibly by following these [ASF guidelines](https://www.apache.org/security/) for reporting.
The contact email for reporting newly discovered exploits in Solr is <mailto:security@solr.apache.org>.
```

Switching to one line per sentence and cleaning up the first sentence a bit.
